### PR TITLE
Refine item colors and opacities for all states

### PR DESCRIPTION
**Pull request by cbjeukendrup**
Resolves: #9173 
Resolves: #9260

Also fixes some minor issues I discovered with margins around dockable toolbars, which caused that the separator lines in the note input bar did not touch the separator lines between the toolbars, leaving redundant 2px gaps.

### CODE REVIEW

**Comment by DmitryArefiev** [comment]
Checked #9173, #9260 on Win10, Mac11.13.1, Linux Mint 20.2 - FIXED

Please merge into master

File: src/instrumentsscene/qml/MuseScore/InstrumentsScene/internal/InstrumentsTreeItemDelegate.qml [hunk 1]
```diff
@@ -333,6 +352,10 @@ Item {
 
                     return ui.theme.bodyFont
                 }
+
+                Component.onCompleted: {
+                    console.log(mapToGlobal(Qt.point(0, 0)))
```
Comment (RomanPudashkin) [inline src/instrumentsscene/qml/MuseScore/InstrumentsScene/internal/InstrumentsTreeItemDelegate.qml:357]: Please, remove this log

File: src/workspace/qml/MuseScore/Workspace/internal/WorkspacesView.qml [hunk 2]
```diff
@@ -51,28 +51,28 @@ RadioButtonGroup {
...
 
+        width: ListView.view.width
         height: 46
+
+        leftPadding: root.leftPadding
         spacing: 12
 
-        text: model.name
-        font: model.isSelected ? ui.theme.bodyBoldFont : ui.theme.bodyFont
+        text: modelname
```
Comment (RomanPudashkin) [inline src/workspace/qml/MuseScore/Workspace/internal/WorkspacesView.qml:62]: Something went wrong here... Probably you should revert these changes

File: src/instrumentsscene/qml/MuseScore/InstrumentsScene/internal/InstrumentsTreeItemDelegate.qml (hunk 1 above)
Comment (cbjeukendrup) [reply src/instrumentsscene/qml/MuseScore/InstrumentsScene/internal/InstrumentsTreeItemDelegate.qml]: Oops!
Thanks, fixed

File: src/workspace/qml/MuseScore/Workspace/internal/WorkspacesView.qml (hunk 2 above)
Comment (cbjeukendrup) [reply src/workspace/qml/MuseScore/Workspace/internal/WorkspacesView.qml]: Very strange issue! 
Anyway, thanks!
Fixed.

**Review by RomanPudashkin — APPROVED** [review]
